app/eth2wrap/eth2wrap_gen.go

```diff
@@ -580,3 +582,21 @@ func (m multi) GenesisTime(ctx context.Context) (time.Time, error) {
 
 	return res0, err
 }
+
+func (m multi) SubmitBeaconCommitteeSubscriptions(ctx context.Context, subscriptions []*eth2exp.BeaconCommitteeSubscription) ([]eth2exp.BeaconCommitteeSubscriptionResponse, error) {
```

Comment:
this file has a generated so you cannot manually edit it. Move this method to `eth2wrap.go`

Reply:
done